src/pages/AboutPage.tsx

```diff
@@ -1,23 +1,55 @@
-import React, { useEffect, useState } from 'react'
+import React from 'react'
+import styles from './styles/AboutPages.module.scss'
+import close from '../assets/close.svg'
 import { Link, RouteComponentProps, withRouter } from 'react-router-dom'
+import PersonalCard from '../components/molecules/PersonalCard/index'
 
-const AboutPage: React.FC<any & RouteComponentProps<any>> = props => {
-  const [message, setMessage] = useState<string>('')
-
-  useEffect(() => {
-    const number = props.match.params.number
-
-    if (number) {
-      setMessage(`El parámetro es ${number}`)
-    } else {
-      setMessage('Sin parámetro!')
-    }
-  }, [props])
+interface Props {
+  className?: string;
+}
 
+const AboutPage: React.FC<any & RouteComponentProps<any>> = Props => {
```

Comment:
Holi! aca no hace falta tipar si o si, ya que Typescript puede asumir que va a hacer un componente de React. Lo de RouteComponentProps seguro estaba como un ejemplo, pero tambien se puede quitar 😄 

Se puede dejar asi:
```suggestion
const AboutPage = () => {
```
o asi con tipo
```suggestion
const AboutPage: React.FC = () => {
```